### DISCUSSION TITLE
Does MPICH support RDMA or RoCE?

### DISCUSSION THREAD

**Discussion by yangbinken** [pmodels/mpich#5935] [Q&A]
Actrually, I'm a beginner for MPICH. 
I have two computers direct-connected with Mellanox NIC(Network Interface Card) and the two NICs has RoCE function. I use default setting to install MPICH 3.2.1, but I can figure out it's faster than using normal NIC. 
Does MPICH support RDMA or RoCE? How can I configure the MPICH to support RDMA or RoCE?

**Comment by hzhou** [comment]
Could you try downloading the latest MPICH release (https://www.mpich.org/downloads/) and configure with --
```
./configure --with-device=ch4:ucx
```

**Reply by niyazmurshed** [reply]
did this work?
